Code/GraphMol/Bond.cpp

```diff
@@ -183,6 +183,9 @@ double Bond::getBondTypeAsDouble() const {
 }
 
 double Bond::getValenceContrib(const Atom *atom) const {
+  if (atom != getBeginAtom() && atom != getEndAtom()) {
```

Comment:
Should this be a warning?  i.e. atom not in bond?

Reply:
not sure... it's unambiguous that the answer is zero.

Reply:
Now that I believe...  I think we can table it for now, I might throw an assertion in for kicks just to see if this every happens internally.